### PR TITLE
[CDAP-7448] - Updates readme + Splits webpack build process

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Splits webpack build process to not build common library along with cdap & login
- Adds livereload to cdap webpack
- Fixes README.md file based on new updates to UI process.

Bamboo build - http://builds.cask.co/browse/CDAP-DRC4727-2

### CODE REVIEW

**Comment by awholegunch** [comment]
I made numerous changes to the README. Please review my changes. LGTM.

File: cdap-ui/app/cdap/cdap.html [hunk 1]
```diff
@@ -31,5 +31,6 @@
   <script type="text/javascript" src="/ui-config.js"></script>
   <script type="text/javascript" src="/cdap_assets/common.js"></script>
   <script type="text/javascript" src="/cdap_assets/cdap.js"></script>
+  <script src="http://localhost:35729/livereload.js"></script>
```
Comment (elfenheart) [inline cdap-ui/app/cdap/cdap.html:34]: Is it hard to make this into optional for dev vs prod build?

File: cdap-ui/package.json [hunk 2]
```diff
@@ -72,7 +68,8 @@
     "main-bower-files": "2.11.1",
     "merge-stream": "1.0.0",
     "style-loader": "0.13.1",
-    "url-loader": "0.5.7"
+    "url-loader": "0.5.7",
+    "webpack-livereload-plugin": "^0.9.0"
```
Comment (elfenheart) [inline cdap-ui/package.json]: specific version?

File: cdap-ui/app/cdap/cdap.html (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/cdap.html:34]: Have filed a JIRA for it (https://issues.cask.co/browse/CDAP-7552). Will take it up in subsequent PRs.

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**